.github/workflow_scripts/lint_check.sh

```diff
@@ -17,4 +17,7 @@ pylint --rcfile=./tests/lint/pylintrc ./python/graphstorm/tracker/
 pylint --rcfile=./tests/lint/pylintrc ./python/graphstorm/run/
 pylint --rcfile=./tests/lint/pylintrc ./python/graphstorm/utils.py
 
-pylint --rcfile=./tests/lint/pylintrc ./python/graphstorm/sagemaker/
\ No newline at end of file
+pylint --rcfile=./tests/lint/pylintrc ./python/graphstorm/sagemaker/
+
+pylint --rcfile=./tests/lint/pylintrc ./python/graphstorm/graphpartitioning/
+pylint --rcfile=./tests/lint/pylintrc ./python/graphstorm/graphpartitioning/array_readwriter/
```

Comment:
do you need to add both?

Reply:
This is indeed not needed. line 23 is now deleted.